test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities
+./test_scripts/API/RAI/002_ATF_N_MissedAllParamaters_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_missing_all_parameters

```

Comment:
@OVikhrov, this is the idea of test. When SDL receives such message from HMI it rejects it and we test case when HMI doesn't respond to UI.GetCapabilities.
According to requirement, case: "SDL does NOT receive value of "steeringWeelLocation" parameter via UI.GetCapabilities_response from HMI"